src/main/scala/devices/pinctrl/PinCtrl.scala

```diff
@@ -13,6 +13,73 @@ class PinCtrl extends Bundle {
   val ie   = Bool()
 }
 
+class LN14LPPEnhancedPinCtrl extends EnhancedPinCtrl {
```

Comment:
What are ps, po, poe, and pe? Is there a way we can add parameterization to EnhancedPinCtrl to add these signals? For example, I think ds0/ds1 could be added by parameterizing the width of the existing ds signal

Reply:
Those are the additional IO signals for GPIO.  EnhancedPinCtrl is used few other places, and it's missing many signal comparing to current GPIO used in Chronos.